### PR TITLE
CIVIL-1388 create a jwt claim persister

### PR DESCRIPTION
**Pull request by walfly**

### CODE REVIEW

File: pkg/claimsstore/jwtclaimpersister.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+func (p *JWTClaimPGPersister) AddJWT(tokenString string, senderDID *didlib.DID) (string, error) {
+	token, err := p.didJWTService.ParseJWT(tokenString)
+	if err != nil {
+		return "", err
+	}
+	hash, err := hashJWT(tokenString)
+	if err != nil {
+		return "", err
+	}
+	fmt.Printf("\n\n\nclaims: %v\n\n\n", token.Claims)
```
Comment (petersng) [inline pkg/claimsstore/jwtclaimpersister.go]: is this for debug?  should it be removed?

File: pkg/claimsstore/jwtclaimpersister.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+		db:            db,
+		didJWTService: didJWTService,
+	}
+}
+
+// AddJWT adds a new jwt claim to the db
+func (p *JWTClaimPGPersister) AddJWT(tokenString string, senderDID *didlib.DID) (string, error) {
+	token, err := p.didJWTService.ParseJWT(tokenString)
+	if err != nil {
+		return "", err
```
Comment (petersng) [inline pkg/claimsstore/jwtclaimpersister.go]: Do we need to wrap some of these errors?

File: pkg/claimsstore/jwtclaimpersister.go (hunk 1 above)
Comment (walfly) [reply pkg/claimsstore/jwtclaimpersister.go]: yep

File: pkg/claimsstore/jwtclaimpersister.go (hunk 2 above)
Comment (walfly) [reply pkg/claimsstore/jwtclaimpersister.go]: will do

**Pull request merged**